openlibrary/plugins/openlibrary/js/SearchBar.js

```diff
@@ -0,0 +1,244 @@
+import { debounce } from './nonjquery_utils.js';
+import * as SearchUtils from './SearchUtils';
+
+const RENDER_INSTANT_SEARCH_RESULT = {
+    books(work) {
+        const author_name = work.author_name ? work.author_name[0] : '';
+        $('header#header-bar .search-component ul.search-results').append(
```

Comment:
See comment on other file - let's get the templates separated from the JS.

Reply:
Closing as duplicate (i.e. WON'T FIX); Don't worry about saying the same thing through different commits; I'll look through all your comments (even if they're on outdated code) :)